mountpoint-s3/src/inode.rs

```diff
@@ -2522,11 +2527,25 @@ mod tests {
         let stat = inode.get_inode_state().unwrap().stat.clone();
         assert!(!stat.is_valid());
 
-        // Should get an error back when calling setattr
+        // Should be able to reset expiry back and make stat valid when calling setattr
```

Comment:
Let's review this test, when/if we change the way the stats are updated in `update_slow_path`.

Reply:
Should I also add a test for lookup on local inode with Invalid InodeStat?